common/consts.go

```diff
@@ -2,14 +2,19 @@ package common
 
 import "time"
 
-const ObserverMaxBlockNumber = 10000
-const ObserverPruneInterval = 10 * time.Minute
-const ObserverAlertInterval = 5 * time.Minute
+const (
+	ObserverMaxBlockNumber = 10000
+	ObserverPruneInterval  = 10 * time.Minute
+	ObserverAlertInterval  = 5 * time.Minute
 
-const DeputyConfirmTxInterval = 5 * time.Second
-const DeputySendTxInterval = 5 * time.Second
-const DeputyCheckTxSentInterval = 5 * time.Second
-const DeputyExpireUserHTLTInterval = 10 * time.Second
-const DeputyAlertInterval = 5 * time.Minute
-const DeputyReconInterval = 1 * time.Hour
-const DeputyMetricsInterval = 10 * time.Second
+	DeputyConfirmTxInterval      = 5 * time.Second
+	DeputySendTxInterval         = 5 * time.Second
+	DeputyCheckTxSentInterval    = 5 * time.Second
+	DeputyExpireUserHTLTInterval = 10 * time.Second
+	DeputyAlertInterval          = 5 * time.Minute
+	DeputyReconInterval          = 1 * time.Hour
+	DeputyMetricsInterval        = 10 * time.Second
+	// Overflow interval must be long compared to the time it takes for a tx to be confirmed.
+	// Otherwise it funds could be sent from the deputy while a previous tx is still processing, resulting in too much being sent out of the account.
+	DeputyRunOverflowInterval = 15 * time.Second // 10 * time.Minute
```

Comment:
I recommend that we use a longer interval that we can accept for transferring funds out of the deputy account like one day. The advantage is it can prevent transferring a small number of tokens frequently. But we can set another threshold like 2 times the current threshold, when the amount is larger than the new threshold, we transfer the funds out immediately.

Reply:
How about `DeputyRunOverflowInterval = 1 * time.Hour`?

I like your suggestion. I can implement it in a future PR.